icu4c/source/test/intltest/plurults.cpp

```diff
@@ -447,6 +450,129 @@ void PluralRulesTest::testGetSamples() {
   }
 }
 
+void PluralRulesTest::testGetFixedDecimalSamples() {
+  // TODO: fix samples, re-enable this test.
+
+  // no get functional equivalent API in ICU4C, so just
+  // test every locale...
+  UErrorCode status = U_ZERO_ERROR;
```

Comment:
Does this file generally use an indent of 2? ICU standard is 4. Please fix unless this file is overwhelmingly different already.